rootfs/Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM quay.io/deis/base:v0.3.6
+FROM minio/mc:RELEASE.2018-11-06T01-12-20Z as mc
 
-RUN adduser --system \
-	--shell /bin/bash \
-	--disabled-password \
-	--home /home/minio \
-	--group \
-	minio
+FROM minio/minio:RELEASE.2018-11-06T01-01-02Z
+
+COPY --from=mc /usr/bin/mc /usr/bin/mc
+
+RUN adduser \
+       -s /bin/sh \
```

Comment:
Same thing here, can we keep using the more verbose `adduser` flags as it will save time if someone does not know what each flag does (--shell, --disabled-password, --home ...). I think this verbosity is good as it is verbosity for clarity at no cost. :+1:

Reply:
I'm sorry about this, because I also like to have clear parameters. But the shell contained by busybox is very limited, and it does not support it.

`
Usage: adduser [OPTIONS] USER [GROUP]

Create new user, or add USER to GROUP

	-h DIR		Home directory
	-g GECOS	GECOS field
	-s SHELL	Login shell
	-G GRP		Group
	-S		Create a system user
	-D		Don't assign a password
	-H		Don't create home directory
	-u UID		User id
	-k SKEL		Skeleton directory (/etc/skel)
`

Reply:
If that is the case, then that is okay and a limitation we can live with.